.github/workflows/CI.yml

```diff
@@ -0,0 +1,115 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'master'
+      - "maintenance/.+"
+  pull_request:
+    branches:
+      - 'master'
+      - "maintenance/.+"
+  schedule:
+    # Nightly tests run on master by default:
+    #   Scheduled workflows run on the latest commit on the default or base branch.
+    #   (from https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule)
+    - cron: '0 0 * * *'
+
+jobs:
+  test:
+
+    runs-on: ${{ matrix.cfg.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+          # Uncomment when ready ;)
+          # - 3.8
+        cfg:
+          - os: ubuntu-latest
+            rdkit: true
+            openeye: false
+
+          - os: ubuntu-latest
+            rdkit: false
+            openeye: true
+
+          - os: ubuntu-latest
+            rdkit: true
+            openeye: true
+
+          - os: macOS-latest
+            rdkit: true
+            openeye: false
+
+    env:
+      ORGNAME: omnia
+      PACKAGENAME: openforcefield
+      OPENEYE_CHANNEL: -i https://pypi.anaconda.org/OpenEye/simple
+      PIP_ARGS: "-U"
```

Comment:
possibly also unused?

Reply:
Let's see if it breaks if I remove it :D